airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -22,27 +22,56 @@
 # SOFTWARE.
 #
 
+import os
 import urllib.parse as urlparse
 from abc import ABC, abstractmethod
 from typing import Any, Iterable, Mapping, MutableMapping, Optional
 from urllib.parse import parse_qs
 
 import pendulum
 import requests
+import vcr
```

Comment:
`vcr` is a default library? dont need to add to setup.py?

Reply:
Oops, I've added it now.